media/fauna-logo.svg

```diff
@@ -1,3 +1,17 @@
-<svg width="19" height="23" viewBox="0 0 19 23" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path d="M1.90219 8.46692L0 0L11.7192 9.15284L13.7089 6.62349L16.2451 6.13048L19 5.59459L16.2451 9.75303V15.3476L2.14269 23L6.01266 16.9338L9.64212 14.7474L5.85961 15.6263L2.60184 10.2246L9.35788 12.0466" fill="#ffffff"/>
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 25.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```

Comment:
Can we remove these two lines safely?

Reply:
I think so, it looks like those are just excess generated by Illustrator. Not 100% sure though.

Reply:
@danieltodonnell Line 2 can go since it's a comment. Looks like we can remove line 1 as well: https://stackoverflow.com/questions/38169475/svg-in-html5-when-is-xml-declaration-xml-version-1-0-encoding-utf-8

Reply:
I removed them and pushed another commit onto your branch 👍